### PR TITLE
Chinese translation samples for index on DevTools

### PR DESCRIPTION
**Pull request by alphayang**

### CODE REVIEW

File: src/content/zh-cn/tools/chrome-devtools/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+* 在Chrome菜单中选择 **更多工具** > **开发者工具**
+* 在页面元素上右键点击，选择 “检查”
+* 使用 [快捷键 ](/web/tools/chrome-devtools/inspect-styles/shortcuts)
+<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Windows) 或 <kbd>Cmd</kbd>+<kbd>Opt</kbd>+<kbd>I</kbd> (Mac)
+
+## 了解面板
+
+### 设备模式
+<img src="/web/tools/chrome-devtools/images/devicemode.png" alt="Device Mode" class="attempt-right">
+使用设备模式构建完全响应式，移动优先的网络体验。</p>
```
Comment (petele) [inline src/content/zh-cn/tools/chrome-devtools/index.md]: Remove extra `</p>`

File: src/content/zh-cn/tools/chrome-devtools/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+description: Option 1
+
+{# wf_updated_on: 2016-07-26 #}
+{# wf_published_on: 2016-03-28 #}
+
+# Chrome 开发者工具 {: .page-title }
+
+Chrome 开发者工具是一套内置于Google Chrome中的Web开发和调试工具，可用来对网站进行迭代、调试和分析。
+
+提示: 寻找最新版本的Chrome 开发者工具, [Chrome Canary](https://www.google.com/intl/en/chrome/browser/canary.html) 总是有最新的DevTools.
```
Comment (petele) [inline src/content/zh-cn/tools/chrome-devtools/index.md]: Please change `提示:` to `Dogfood:`

File: src/content/zh-cn/tools/chrome-devtools/index.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Option 1
+
+{# wf_updated_on: 2016-07-26 #}
```
Comment (petele) [inline src/content/zh-cn/tools/chrome-devtools/index.md]: Please change `wf_updated_on` to the date you translated this.

File: src/content/zh-cn/tools/chrome-devtools/index.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Option 1
```
Comment (petele) [inline src/content/zh-cn/tools/chrome-devtools/index.md]: Be sure to set the correct description.

File: src/content/zh-cn/tools/chrome-devtools/index.md (hunk 2 above)
Comment (alphayang) [inline src/content/zh-cn/tools/chrome-devtools/index.md]: Done

**Pull request merged**